etc/views.sql

```diff
@@ -39,65 +41,37 @@ ON
 	p.wd_id = w.cldf_id
 GROUP BY p.u_id;
 
-
--- Utterance information
-DROP VIEW IF EXISTS phonestats;
-CREATE VIEW phonestats AS
+-- Number of phones per word
+DROP VIEW IF EXISTS phones_per_word;
+CREATE VIEW phones_per_word AS
 SELECT
-    w.cldf_id AS wd_id,
-	count(p.wd_id) AS num_phones,
-	w.cldf_languageReference
+    wd_id,
+	count(cldf_id) AS num_phones
```

Comment:
@Tarotis I tried to standardize the nomenclature a bit, using `num_` whenever things are counted, and `_per_` whenever things are grouped. I'm not sure how transparent that is, but at least we don't have `num_phones` and `WordCount` in the same SQL file anymore :)